tests/aws/services/cloudformation/api/test_stacks.py

```diff
@@ -878,9 +878,8 @@ def test_name_conflicts(aws_client, snapshot, cleanups):
 
 @markers.aws.validated
 def test_describe_stack_events_errors(aws_client, snapshot):
-    with pytest.raises(aws_client.cloudformation.exceptions.ClientError) as e:
+    with pytest.raises(botocore.exceptions.ParamValidationError) as e:
         aws_client.cloudformation.describe_stack_events()
-    snapshot.match("describe_stack_events_no_stack_name", e.value.response)
```

Comment:
@silv-io: I feel like it is a bit weird to assert on client side errors, we're effectively testing Botocore behavior here. 

It might have been better to disable client side validation to keep the snapshot test and verify that for SDKs that do not implement this validation, we still raise the proper error?

Not a big deal but saw this in my missed PR reviews 😄